test/rmath.jl

```diff
@@ -185,8 +185,8 @@ rmathcomp_tests("norm", [
     ((0.0, 1.0), -6.0:0.01:6.0),
     ((2.0, 1.0), -3.0:0.01:7.0),
     ((0.0, 0.5), -3.0:0.01:3.0),
-    ((0, 1), -3:3),
-    ((0, 1), -Float16(3):Float16(0.01):Float16(3))
```

Comment:
I see. I've opened https://github.com/JuliaStats/StatsFuns.jl/pull/26. When that has been merged, I'll move the tag and make it a minor.